### PR TITLE
Improve office world

### LINKED ISSUE #60: [Meta] Improve office world
### Summary

image

Improve overall world
 - Add a floor
 - Add textures to the walls
 - Add office assets
 - Add lights

### Suggestions
 - Let's work on blender. 
 - Use current stl as guidelines for the walls
 - Use objects for different parts of the office:
   - Walls:
   - Wooden Floor 
   - Tile Floor
 - Textures
   - Apply pbr textures (at least albedo, normals and roughness)
     - For walls: All white for the moment is ok.
     - Size: Let's keep size of textures <10mb 

### Labor breakdown
Blender
 - [x] Object1: Base world creation with only walls. No textures applied.
 - [x] Object2: Add tile floor to the entire space. No textures applied
 - [x] Object3: Add wooden floor just to the wooden floor area. No textures applied.
 - [ ] Object4: Add basic roof
      Normals of the mesh pointing down would allows us to see it from above in the sim :smile: 
 - [x] Apply textures to walls.
 - [x] Apply textures to tile floor.
 - [x] Apply texture to wooden floor.
 - [x] Export to a office.dae: walls, tile_floor and wooden_floor will be submeshes

Gazebo
 - [x] Create office model and costume the dae files and apply PBR textures
 - [x] Create sdf world and use the office model.
 - [x] Create a office_populated world that consumes the same office model but add some office assets in the world.
 - [ ] Add correct lightning. There is no need for visual lamp assets for the first iteration of it.

### PR DESCRIPTION
**Pull request by larodriguez22**
# 🎉 New feature

Related to #60 

## Summary
Improvement of office world by using blender and pbr textures.

[Screencast from 08-19-2024 05:09:46 PM.webm](https://github.com/user-attachments/assets/2fdb737c-26fc-4b18-9911-c8a30aca857c)

## Test it
To run:
```
ros2 launch andino_gz andino_gz.launch.py world_name:=office.sdf
```

### CODE REVIEW

File: andino_gz/models/office/model.config [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+
+<model>
+  <name>office</name>
+  <version>1.0</version>
+  <sdf version="1.6">model.sdf</sdf>
+
+  <author>
+    <name>Gazebo</name>
```
Comment (francocipollone) [inline andino_gz/models/office/model.config]: nit: Add your name!

File: andino_gz/models/office/model.config [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  </author>
+
+  <description>
+    Office model
+  </description>
+
+</model>
```
Comment (francocipollone) [inline andino_gz/models/office/model.config]: nit: Add missing empty line at the end

File: andino_gz/models/office/model.sdf [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+              <metalness>0.</metalness>
+            </metal>
+          </pbr>
+        </material>
+      </visual>
+    </link>
+    
+    <pose>0 0 0 0 0 0</pose>
+  </model>
+</sdf>
```
Comment (francocipollone) [inline andino_gz/models/office/model.sdf]: nit: Add missing empty line at the end

File: andino_gz/worlds/office.sdf [hunk 4]
```diff
@@ -1,6 +1,6 @@
 <?xml version='1.0' encoding='ASCII'?>
 <sdf version='1.7'>
-  <world name='gazebo_world'>
```
Comment (francocipollone) [inline andino_gz/worlds/office.sdf]: actually remove this change. We need it to be named gazebo_world due to some logic later on when having multiple robots

**Review by francocipollone — CHANGES_REQUESTED** [review]
Looks really good! :rocket: 
I will try it locally!

File: andino_gz/models/office/model.config (hunk 1 above)
Comment (larodriguez22) [reply andino_gz/models/office/model.config]: done

File: andino_gz/models/office/model.config (hunk 2 above)
Comment (larodriguez22) [reply andino_gz/models/office/model.config]: done

File: andino_gz/models/office/model.sdf (hunk 3 above)
Comment (larodriguez22) [reply andino_gz/models/office/model.sdf]: done

File: andino_gz/worlds/office.sdf (hunk 4 above)
Comment (larodriguez22) [reply andino_gz/worlds/office.sdf]: done

**Comment by francocipollone** [comment]
Before merging try to fix DCO: https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md for the commits.

**Review by francocipollone — DISMISSED** [review]
It is working really good even with the old map's png for navigation

image

One minor comment: I think you are missing one wooden hallway
image

**Comment by francocipollone** [comment]
Given that you are going to modify a bit the wooden floor, can you check that actually the wooden hallway should be a bit shorter:

For example
image

You can see this in this videos as well [click here for video](https://private-user-images.githubusercontent.com/53065142/279692835-283f4afd-0f9a-4d37-b71f-c9d7b2f3e453.mp4?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQzMDM2OTEsIm5iZiI6MTcyNDMwMzM5MSwicGF0aCI6Ii81MzA2NTE0Mi8yNzk2OTI4MzUtMjgzZjRhZmQtMGY5YS00ZDM3LWI3MWYtYzlkN2IyZjNlNDUzLm1wND9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODIyVDA1MDk1MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWZmNzkyZjczNzAxNjc3MzQzMjRiNTFmMzZhYThhZWFhY2NiMzllYmU2ZjRlZmM5YTE3MDI3NTA3ZGQ0OWJmOTcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.g2y3pXM8eGbo3ThIgoBvUy3qeXYtx4yisKhJwyRX3wo)

**Comment by larodriguez22** [comment]
> Before merging try to fix DCO: https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md for the commits.

Fixed

**Review by Voldivh — APPROVED** [review]
LGTM! Awesome job, we needed to make some changes to the andino_rmf repo, we should try to merge this PR and https://github.com/Ekumen-OS/andino_rmf/pull/27 at the same time.

**Pull request merged**